.github/workflows/build.yml

```diff
@@ -0,0 +1,103 @@
+          Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        key: ${{ runner.os }}-${{ inputs.ubuntu-version }}-${{ inputs.godot-version }}
+    - name: Download Godot and its Export Templates
+      if: steps.cache-godot.outputs.cache-hit != 'true'
+      run: |
+        wget -q https://downloads.tuxfamily.org/godotengine/${{ inputs.godot-version }}/Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        wget -q https://downloads.tuxfamily.org/godotengine/${{ inputs.godot-version }}/Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+    - name: Add dependencies to the cache if necessary
+      if: steps.cache-godot.outputs.cache-hit != 'true'
+      uses: actions/cache/save@v4
+      with:
+        path: |
+          Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+          Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        key: ${{ steps.cache-godot.outputs.cache-primary-key }}
+    - name: Install Godot and the export templates
+      run: |
+        unzip Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        rm Godot_v${{ inputs.godot-version }}-stable_linux_headless.64.zip
+        unzip Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        rm Godot_v${{ inputs.godot-version }}-stable_export_templates.tpz
+        mkdir -v -p ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+        mv templates/* ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+        ls ~/.local/share/godot/templates/${{ inputs.godot-version }}.stable/
+    - name: Install mingw
+      run: sudo apt-get install gcc-mingw-w64
+    - name: Create the output directories
+      run: |
+        mkdir output
+        mkdir output/burrito_link
+    - name: Build X11_FG
+      run: |
+        cd burrito-fg
+        cargo build --release
+    - name: Build taco_parser
+      run: |
+        cd taco_parser
+        cargo build --release
+    - name: Build Burrito Link
+      run: |
+        mkdir burrito_link/build
+        cd burrito_link/build
+        cmake ..
+        make
+    - name: Build Burrito
+      run: |
+        mkdir build
+        ./Godot_v${{ inputs.godot-version }}-stable_linux_headless.64 --export "Linux/X11"
+        chmod +x build/burrito.x86_64
+    - name: Fill the output folder
+      run: |
+        mv burrito_link/build/burrito_link.exe output/burrito_link
+        mv burrito_link/build/d3d11.dll output/burrito_link
+        mv build/burrito.x86_64 output/
+        mv build/libburrito_fg.so output/
+        mv build/libgw2_taco_parser.so output/
+    - name: Upload output folder as an artifact
+      uses: actions/upload-artifact@v4
+      id: artifact-upload
+      with:
+        name: burrito-binaries
+        path: output
+        if-no-files-found: error
+        overwrite: true
+    - name: Strip the burrito_link binaries for reproducibility
+      run: strip output/burrito_link/*
```

Comment:
The artifact we upload (to use in the release) is uploaded before this step.

The stripped binaries are deleted at the end of the job.

Reply:
You are totally right.
Because they are being stripped in-place I think we should add a copy to duplicate these files into a new temporary location before stripping them in-place. If for no other reason then it is completely obvious that we are not stripping the output builds.

Reply:
I do not understand who we are doing this for.
Do you see someone looking through our CI logs, going like `Oh they are stripping their binaries I wonder if the release contains the stripped ones or not`? I do not think anyone cares.

That said, if you insist I will change it, but I can not follow your reasoning.

Reply:
If you would rather: I am also ok with combining the last two steps here and adding a `rm` to delete all of the files in output after this step.

The main thing I want avoid is silently changing the release process to release a stripped binary. Which could easily happen if someone adds a new build step to the end of this file, notices their new artifact is not present in the build artifacts, moves the artifact upload step to the end of the file.

If we delete the stripped binary before any other step has a chance to use it then it will be obvious in the build artifacts because the binary is not there.

> Oh they are stripping their binaries I wonder if the release contains the stripped ones or not

More like "I am trying to debug something for a user and their binary has been stripped, now debugging is harder". I dont think anyone besides the person doing debugging cares about debug symbols.